.travis.yml

```diff
@@ -0,0 +1,55 @@
+language: minimal
+branches:
+   only:
+     - master
+env:
+  global:
+    - ORG='activiti'
+    - APP_NAME='activiti-cloud-full-example'
+    - PREVIEW_NAMESPACE="example-$TRAVIS_BRANCH-$TRAVIS_BUILD_NUMBER";
```

Comment:
TRAVIS_BRANCH is the target PR branch, not the source one, maybe you should TRAVIS_PULL_REQUEST_BRANCH instead?

Reply:
this is temporary namespace and it is run also on master TRAVIS_BRANCH works ok without additional configuration